.github/workflows/ci.yml

```diff
@@ -0,0 +1,62 @@
+name: CI
+'on':
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+jobs:
+  # lint-dockerfile:
+  #   name: Lint Dockerfile
+  #   runs-on: ubuntu-latest
+  #   steps:
+  #     - name: Git checkout
+  #       uses: actions/checkout@v2
+  #     - name: Run Hadolint Dockerfile Linter (Common)
+  #       uses: burdzwastaken/hadolint-action@master
+  #       env:
+  #         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  #         HADOLINT_ACTION_DOCKERFILE_FOLDER: docker/cygnus-common
+  #     - name: Run Hadolint Dockerfile Linter (NGSI)
+  #       uses: burdzwastaken/hadolint-action@master
+  #       env:
+  #         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  #         HADOLINT_ACTION_DOCKERFILE_FOLDER: docker/cygnus-ngsi
+  #     - name: Run Hadolint Dockerfile Linter (NGSI-LD)
+  #       uses: burdzwastaken/hadolint-action@master
+  #       env:
+  #         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  #         HADOLINT_ACTION_DOCKERFILE_FOLDER: docker/cygnus-ngsi-ld
+  #     - name: Run Hadolint Dockerfile Linter (Twitter)
+  #       uses: burdzwastaken/hadolint-action@master
+  #       env:
+  #         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  #         HADOLINT_ACTION_DOCKERFILE_FOLDER: docker/cygnus-twitter
+  
+  unit-test:
+    name: Unit Tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Git checkout
+        uses: actions/checkout@v2
+      - name: Use Java 8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 8
```

Comment:
In a similar line to https://github.com/telefonicaid/perseo-core/pull/167/files#r534058833, Java 9, 10 and 11 should be included here, although in commented form (with the same remark used in travis: `FIXME: something in the current build (pom.xml?) precludes Java>8 to work`).